core/tests/image-cache-data-url.test.ts

```
@@ -0,0 +1,101 @@
+
+let sharedMockDb = {
+  getData: vi.fn().mockReturnValue({}),
+  set: vi.fn(),
+  update: vi.fn(),
+  commit: vi.fn(),
+};
+
+vi.mock("../src/utils", () => ({
+  JsonDB: class MockJsonDB {
+    constructor() {
+      return sharedMockDb;
```

Biome (2.4.6)

[error] The constructor should not return a value.

(lint/correctness/noConstructorReturn)